lib/tasks/mutant.rake

```diff
@@ -0,0 +1,49 @@
+task :mutant do
+  classes_to_mutate.each do |klass|
+    vars = 'NOCOVERAGE=true'
+    flags = '--include lib --use rspec'
+    unless system("#{vars} mutant #{flags} #{klass}")
+      raise 'Mutation testing failed'
+    end
+  end
+end
+
+task(:default).prerequisites << task(:mutant)
+
+private
+
+def classes_to_mutate
+  files = grep_files_for_classes
+  klasses = extract_classes_for_mutation(files)
+  klasses.map { |k|
+    setup_class_for_run(k)
+  }
+end
+
+# This is a nasty hack because we’re using POROs; otherwise, we could just use
```

Comment:
This comment needs to be tweaked, because it's got some Rails-specific advice (ApplicationRecord.descendants) which isn't relevant for this project.